<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="font">
    <font/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">QWidget#centralWidget{
border-radius:40px;
}</string>
   </property>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>900</width>
      <height>600</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="loginPage">
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>900</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#mainWidget{
background-color: rgb(255,255,255);
border-radius:40px;
}</string>
      </property>
      <widget class="QWidget" name="signupWidget" native="true">
       <property name="geometry">
        <rect>
         <x>548</x>
         <y>2</y>
         <width>350</width>
         <height>596</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#signupWidget{
background-color: rgb(31,31,31);
border-top-right-radius:40px;
border-bottom-right-radius:40px;
}</string>
       </property>
       <widget class="QPushButton" name="closeButton_2">
        <property name="geometry">
         <rect>
          <x>288</x>
          <y>30</y>
          <width>28</width>
          <height>28</height>
         </rect>
        </property>
        <property name="mouseTracking">
         <bool>false</bool>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#closeButton_2{
	image:url(:/new/prefix1/icons/close_2.png);
}

QWidget#closeButton_2:pressed{
	image:url(:/new/prefix1/icons/close_pressed.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="mainLabel_4">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>140</y>
          <width>250</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>33</pointsize>
          <bold>true</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255);
</string>
        </property>
        <property name="text">
         <string>New Here ?</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QLabel" name="mainLabel_5">
        <property name="geometry">
         <rect>
          <x>25</x>
          <y>220</y>
          <width>300</width>
          <height>161</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Poppins</family>
          <pointsize>17</pointsize>
          <bold>false</bold>
          <stylestrategy>PreferAntialias</stylestrategy>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255,255,255);
</string>
        </property>
        <property name="text">
         <string>Sign up and start saving 
money in the best way!</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
       <widget class="QPushButton" name="signupButton">
        <property name="geometry">
         <rect>
          <x>55</x>
          <y>430</y>
          <width>240</width>
          <height>50</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget#signupButton{
	background-color: rgb(255, 255, 255);
	border-radius:20px;
	color: rgb(35,35,35);
}

QWidget#signupButton:pressed{
	background-color: rgb(170, 170, 170);
	border-radius:20px;
	color: rgb(35,35,35);
}</string>
        </property>
        <property name="text">
         <string>Sign Up</string>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </widget>
      <widget class="QLabel" name="mainLabel_3">
       <property name="geometry">
        <rect>
         <x>74</x>
         <y>100</y>
         <width>400</width>
         <height>111</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>33</pointsize>
         <bold>true</bold>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(31,31,31);
</string>
       </property>
       <property name="text">
        <string>Log In 
to Your Account</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="loginEdit">
       <property name="geometry">
        <rect>
         <x>144</x>
         <y>280</y>
         <width>260</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#loginEdit{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="inputMask">
        <string extracomment="login"/>
       </property>
       <property name="placeholderText">
        <string>username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="passwordEdit">
       <property name="geometry">
        <rect>
         <x>144</x>
         <y>360</y>
         <width>260</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#passwordEdit{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
	color: rgb(255, 255, 255);
}</string>
       </property>
       <property name="inputMask">
        <string/>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="echoMode">
        <enum>QLineEdit::Password</enum>
       </property>
       <property name="placeholderText">
        <string>password</string>
       </property>
      </widget>
      <widget class="QWidget" name="loginWidget" native="true">
       <property name="geometry">
        <rect>
         <x>124</x>
         <y>280</y>
         <width>300</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#loginWidget{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
}</string>
       </property>
      </widget>
      <widget class="QWidget" name="passwordWidget" native="true">
       <property name="geometry">
        <rect>
         <x>124</x>
         <y>360</y>
         <width>300</width>
         <height>50</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#passwordWidget{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
}</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginButton">
       <property name="geometry">
        <rect>
         <x>154</x>
         <y>440</y>
         <width>240</width>
         <height>50</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>12</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QWidget#loginButton{
	background-color: rgb(90, 90, 90);
	border-radius:20px;
	color: rgb(255,255,255);
}

QWidget#loginButton:pressed{
	background-color: rgb(195, 195, 195);
	border-radius:20px;
	color: rgb(255,255,255);
}</string>
       </property>
       <property name="text">
        <string>Sign In</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <zorder>signupWidget</zorder>
      <zorder>mainLabel_3</zorder>
      <zorder>loginWidget</zorder>
      <zorder>loginEdit</zorder>
      <zorder>passwordWidget</zorder>
      <zorder>passwordEdit</zorder>
      <zorder>loginButton</zorder>
     </widget>
    </widget>
    <widget class="QWidget" name="registerPage"/>
    <widget class="QWidget" name="mainPage">
     <property name="font">
      <font>
       <family>Poppins</family>
      </font>
     </property>
     <widget class="QPushButton" name="closeButton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>30</y>
        <width>28</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#closeButton{
	image:url(:/new/prefix1/icons/close.png);
}

QWidget#closeButton:pressed{
	image:url(:/new/prefix1/icons/close_pressed.png);
}</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QWidget" name="leftMenuWidget" native="true">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>-2</x>
        <y>0</y>
        <width>60</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#leftMenuWidget{
background-color: rgb(31,31,31);
border-top-left-radius:40px;
border-bottom-left-radius:40px;
}</string>
      </property>
      <widget class="QPushButton" name="settingsButton">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>530</y>
         <width>28</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image:url(:/new/prefix1/icons/settings.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="accountButton">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>70</y>
         <width>28</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image:url(:/new/prefix1/icons/account.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="chartButton">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>150</y>
         <width>28</width>
         <height>28</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image:url(:/new/prefix1/icons/chart.png)</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mainMenuWidget" native="true">
      <property name="geometry">
       <rect>
        <x>58</x>
        <y>0</y>
        <width>840</width>
        <height>600</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget#mainMenuWidget{
background-color: rgb(255, 255, 255);
border-top-right-radius:40px;
border-bottom-right-radius:40px;
}
</string>
      </property>
      <widget class="QLabel" name="mainLabel">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>301</width>
         <height>91</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>25</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(31,31,31);
</string>
       </property>
       <property name="text">
        <string>Monitor health of  
your financy</string>
       </property>
      </widget>
      <widget class="QLabel" name="mainLabel_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>341</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Poppins</family>
         <pointsize>10</pointsize>
         <stylestrategy>PreferAntialias</stylestrategy>
         <kerning>true</kerning>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(117, 117, 117);
</string>
       </property>
       <property name="text">
        <string>Control and analyze your finances in the easiest way</string>
       </property>
      </widget>
     </widget>
     <zorder>leftMenuWidget</zorder>
     <zorder>mainMenuWidget</zorder>
     <zorder>closeButton</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>853</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>closeButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>849</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>449</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
